pkgs/applications/video/mpv/scripts/occivink.nix

```diff
@@ -1,14 +1,17 @@
 { lib
-, stdenvNoCC
 , fetchFromGitHub
+, buildLua
 }:
 
 let
-  script = { n, ... }@p:
-    stdenvNoCC.mkDerivation (lib.attrsets.recursiveUpdate {
-      pname = "mpv_${n}";
-      passthru.scriptName = "${n}.lua";
-
+  camelToKebab = let
```

Comment:
i find it slight overkill to have this function for single-use here. but maybe it could be moved to lib/strings.nix

Reply:
after irl discussion it has been decided to postpone this to a later point and just merge this as is